src/aead/aead.rs

```diff
@@ -77,11 +77,14 @@ impl OpeningKey {
 
 /// Authenticates and decrypts (“opens”) data in place. When
 ///
-/// The input may have a prefix that is `in_prefix_len` bytes long; any such
+/// `nonce` must be unique for every use of the key to seal data; it must be
```

Comment:
Since this is the `open_in_place` method, maybe the nonce should be explained as "the same nonce as was used for sealing"?